### ISSUE TITLE
README.md - Link to 'Integrations plugin wiki' (https://github.com/mautic-inc/plugin-integrations/wiki)  shows PAGE NOT FOUND

### ISSUE THREAD

**Issue by multikoop** [mautic/plugin-helloworld#10]
Clicking on the link "Integrations plugin wiki" (https://github.com/mautic-inc/plugin-integrations/wiki)  links to a not existent page.
Looks like the link is not valid. Any idea what the new link should be?

**Comment by RCheesley** [comment]
@escopecz any ideas?

**Comment by escopecz** [comment]
I think that @alanhartless was working on some docs for the IntegrationsBundle, but cannot find it.

**Comment by RCheesley** [comment]
That will be in the new developer docs I'm guessing: https://mautic-developer.readthedocs.io/en/latest/plugins/getting_started.html which is still WIP

**Comment by kuzmany** [comment]
That docs was part of wiki in the old repo of integration bundle (in M2 or M3 it was not part of core). 
It was moved  here https://developer.mautic.org/#integration-framework

**Comment by kuzmany** [comment]
Related PR https://github.com/mautic/developer-documentation/pull/183

**Comment by RCheesley** [comment]
Yep - that has been migrated and updated in the new dev docs I believe so we should be OK for the time being.

If anyone would like to help finishing up the dev docs we would love the help!
